tokio-test/src/task.rs

```diff
@@ -91,15 +118,17 @@ impl<T: Unpin> ops::DerefMut for Spawn<T> {
 }
 
 impl<T: Future> Spawn<T> {
-    /// Polls a future
+    /// If `T` is a [`Future`] then poll it. This will handle pinning and the context
+    /// type for the future.
```

Comment:
a little misleading—kinda makes me think that the code would be

`if t.is_future() { poll } else { do nothing }`

Reply:
yeah though not sure how to describe that its a type system if statement rather than a rust language one...